code/game/mecha/combat/tank.dm

```diff
@@ -0,0 +1,135 @@
+#define NOMINAL    0
+#define FIRSTRUN   1
+#define POWER      2
+#define DAMAGE     3
+#define IMAGE      4
+#define WEAPONDOWN 5
+
+/obj/mecha/combat/tank
+	name = "light adhomian tank"
+	desc = "The Ha'rron MK.IV light tank is an armored vehicle commonly used by tajaran military forces."
+	icon = 'icons/mecha/mecha_large.dmi'
+	icon_state = "tank"
+	initial_icon = "tank"
+	pixel_x = -16
+	step_in = 5
+	dir_in = 1
+	health = 800
+	deflect_chance = 60
```

Comment:
does this deflect lazors?

Reply:
Any projectile and melee, it seems. I was nearly sure it was only melee hits, but I will lower this value.